### PR TITLE
Issue 1 simplify set my commands

### PR DESCRIPTION
**Pull request by gerasdf**
Code not needed in `setMyCommands()`

### CODE REVIEW

File: src/UniversalTelegramBot.h [hunk 1]
```diff
@@ -22,16 +22,24 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
 #ifndef UniversalTelegramBot_h
 #define UniversalTelegramBot_h
 
+#define ARDUINOJSON_DECODE_UNICODE 1
+#define ARDUINOJSON_USE_LONG_LONG 1
 #include <Arduino.h>
 #include <ArduinoJson.h>
 #include <Client.h>
+#include <core_version.h>
```
Comment (witnessmenow) [inline src/UniversalTelegramBot.h:30]: what is this?

**Comment by witnessmenow** [comment]
Thanks for the PR @gerasdf 

I left a comment on a new include, do you know where that is from? It seems board specific (I know not all the changes in this PR are yours though)

Could you please add an example for setting the commands, and also please update the readme with information related to it too?

**Comment by witnessmenow** [comment]
Do you have a suggested way to approach these 3 PRs @gerasdf ? It seems like this branch contains the changes of the other PR branch #175

**Comment by gerasdf** [comment]
Sorry it took me so long to answer this.

I don't really know who added that include, it should be coming from something I merged (and I totally missed it, thanks for pointing).

It's on my drive, and it just defines 4 things

```#define ARDUINO_ESP8266_GIT_VER 0x3d128e5c
#define ARDUINO_ESP8266_GIT_DESC 2.6.3
#define ARDUINO_ESP8266_RELEASE_2_6_3
#define ARDUINO_ESP8266_RELEASE "2_6_3"
```

I checked and it's save to remove it.

**Comment by gerasdf** [comment]
I'll do commands example, readme, and I'll remove this include... and then re-submit... though I'm not sure how to do it, heh. Give me a couple of days to try it

**Comment by gerasdf** [comment]
superseded by #180

**Pull request closed without merging**